test/unit/tracks/audio-track.test.js

```diff
@@ -0,0 +1,162 @@
+import AudioTrack from '../../../src/js/tracks/audio-track.js';
+import * as AudioTrackEnums from '../../../src/js/tracks/audio-track-enums.js';
+import TrackBaseline from './track-baseline';
+
+const defaultTech = {
+  audioTracks() {},
+  on() {},
+  off() {},
+  currentTime() {}
+};
+
+q.module('Audio Track', {
+  beforeEach() {
+    this.audioTrackList = [];
+    this.tech = defaultTech;
+    this.tech.audioTracks = () => {
+      return this.audioTrackList;
+    };
+  }
+});
+
+// do baseline track testing
+TrackBaseline(AudioTrack, {
+  id: '1',
+  language: 'en',
+  label: 'English',
+  kind: 'main',
+  tech: defaultTech
+});
+
+test('Can create an AudioTrack an enabled property', function() {
+  let enabled = true;
+  let track = new AudioTrack({
+    enabled,
+    tech: this.tech
+  });
+  equal(track.enabled, enabled, 'enabled value matches what we passed in');
+});
+
+test('defaults when items not provided', function() {
+  let track = new AudioTrack({
+    tech: this.tech
+  });
+
+  equal(track.kind, '', 'kind defaulted to empty string');
+  equal(track.enabled, false, 'enabled defaulted to true since there is one track');
+  equal(track.label, '', 'label defaults to empty string');
+  equal(track.language, '', 'language defaults to empty string');
+  ok(track.id.match(/vjs_audio_track_\d{5}/), 'id defaults to vjs_audio_track_GUID');
+});
+
+test('kind can only be one of several options, defaults to empty string', function() {
+  let track = new AudioTrack({
+    tech: this.tech,
+    kind: 'foo'
+  });
+
+  equal(track.kind, '', 'the kind is set to empty string, not foo');
+  notEqual(track.kind, 'foo', 'the kind is set to empty string, not foo');
```

Comment:
I would vote to skip this level of input validation. I think the number of people who will try to create tracks programmatically is very small and they're going to have to read the docs to understand how to do it anyways.